akumulid/tcp_server.cpp

```diff
@@ -361,7 +362,11 @@ struct TcpServerBuilder {
 
     std::shared_ptr<Server> operator () (std::shared_ptr<DbConnection> con,
                                          std::shared_ptr<ReadOperationBuilder>,
-                                         const ServerSettings& settings) {
+                                         ServerSettings& settings) {
+        if (sysconf(_SC_NPROCESSORS_ONLN) <= 4)
```

Comment:
Why are you using sysconf and std::thread::hardware_concurrency at the same time? And it is better to always use curly braces with if's and other control structures.